client/offchain/src/api.rs

```diff
@@ -16,17 +16,17 @@
 // You should have received a copy of the GNU General Public License
 // along with this program. If not, see <https://www.gnu.org/licenses/>.
 
-use std::{collections::HashSet, str::FromStr, sync::Arc, thread::sleep};
```

Comment:
All the changes in here should also be reverted. The point was about removing the stuff from `im-online` and not everywhere else.